evergreen/7.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# This script runs in a container built from Dockerfile-centos7.
+# It assumes the workspace directory is mounted on /etc/envoy-serverless and the build directory is optionally mount on /build
+
+set -euo pipefail
+set -x
+
+# Use MongoDB toolchain installed in the docker image.
+export LD_LIBRARY_PATH="/opt/mongodbtoolchain/v4/lib"
+export CC=/opt/mongodbtoolchain/v4/bin/clang
+export CXX=/opt/mongodbtoolchain/v4/bin/clang++
+
+# Test targets listed in repo's .bazelci config
+export bazel_tests="//test/common/common/... //test/integration/... //test/exe/..."
+# Include serverless specific patches.
+export bazel_tests="${bazel_tests} //test/extensions/filters/listener/proxy_protocol:proxy_protocol_test //test/extensions/common/proxy_protocol:proxy_protocol_header_test //test/extensions/transport_sockets/proxy_protocol:proxy_protocol_integration_test"
+# This test started failing in 1.17, but only when run by evergreen. not reproducible
```

Comment:
does 1.17 refer to envoy version 1.17? (i know this is copied from sre comments but just curious)

Reply:
Yeah, it refers to Envoy version I believe.